sheets/pg_top

```diff
@@ -10,10 +10,10 @@ pg_top -p 5432 -U postgres -d database -W
 #      Display the actual query plan (EXPLAIN ANALYZE)
 #  E
 #      Display re-determined execution plan (EXPLAIN) of the SQL statement
-#      by a backend process
+#      by a back end process
 #  L
-#      Display the currently held locks by a backend process
+#      Display the currently held locks by a back end process
 #  Q
-#      Display the currently running query of a backend process
+#      Display the currently running query of a back end process
```

Comment:
I'd say back-end, because it's a single description split between two words of a word. Same goes for some other cases in these commits, for lack of the proper grammatical term. If you were to, for example, say "this is the back end", then I wouldn't see a need to add the hyphen.